src/controllers/expenseController.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const userServices = require('../services/userServices');
+const expenseServices = require('../services/expenseServices');
+
+function getAllExpenses(req, res) {
+  const {
+    userId,
+    categories,
+    from,
+    to,
+  } = req.query;
+
+  const filteredExpenses = expenseServices
```

Comment:
Check if `userId` is provided before calling `getExpenses` function to avoid unnecessary function calls and to return a 400 response if `userId` is not provided.

Reply:
Maybe Im doing smthng wrong, but tests fail in that way
![image](https://github.com/mate-academy/node_accounting-app-v2/assets/113321536/6b3b06d6-819b-4444-9be2-7633daf2a1d7)

Reply:
Oh, my but, it's ok